container/src/spec.rs

```diff
@@ -0,0 +1,215 @@
+use std::collections::HashMap;
+use std::path::PathBuf;
+
+use oci_spec::image::ImageConfiguration;
+use oci_spec::runtime::{Process, Spec, SpecBuilder};
+use oci_spec::OciSpecError;
+
+pub const BUNDLE_CONFIG: &str = "config.json";
+
+const ANNOTATION_OS: &str = "org.opencontainers.image.os";
+const ANNOTATION_AUTHOR: &str = "org.opencontainers.image.author";
+const ANNOTATION_CREATED: &str = "org.opencontainers.image.created";
+const ANNOTATION_STOP_SIGNAL: &str = "org.opencontainers.image.stopSignal";
+const ANNOTATION_ARCHITECTURE: &str = "org.opencontainers.image.architecture";
+const ANNOTATION_EXPOSED_PORTS: &str = "org.opencontainers.image.exposedPorts";
```

Comment:
All those annotations are defined in the [`oci_spec`](https://github.com/containers/oci-spec-rs/blob/main/src/image/annotations.rs) crate, please do not redefine them.

Reply:
Should I remove the ones that are not implemented in the [oci_spec](https://github.com/containers/oci-spec-rs/blob/main/src/image/annotations.rs)? And do you think I should implement all the 14 annotations described in the [oci_spec](https://github.com/containers/oci-spec-rs/blob/main/src/image/annotations.rs)?

Reply:
Please keep the `build_annotations` as is, but make it so that it uses the annotation definitions from the `oci_spec`and remove your definitions from the PR. Does that make sense?